Wake On Lan/Modules/AboutScreen/Tests/AboutScreenTests/AboutScreenInteractorTests.swift

```diff
@@ -0,0 +1,36 @@
+import XCTest
+
+@testable import AboutScreen
+@testable import WOLResources
+import WOLResourcesMock
+
+final class AboutScreenInteractorTests: XCTestCase {
+
+    var sut: AboutScreenInteractor!
+    var bundleInfoProviderMock: ProvidesBundleInfoMock!
+    var presenterMock: AboutScreenInteractorOutputMock!
+
+    override func setUp() {
+        super.setUp()
+        bundleInfoProviderMock = ProvidesBundleInfoMock()
+        presenterMock = AboutScreenInteractorOutputMock()
+        sut = AboutScreenInteractor(bundleInfoProvider: bundleInfoProviderMock)
+        sut.presenter = presenterMock
+    }
+
+    func testFetchBundleInfo() {
+        let bundleInfo = BundleInfo(
```

Comment:
The first of all, we should separate each method into several parts:
```
// given
let bundleInfo = BundleInfo()
bundleInfoProviderMock.fetchBundleInfoReturnValue = bundleInfo
// when
sut.fetchBundleInfo()
// then
XCTAssertEqual(presenterMock.interactorDidFetchBundleInfoCallsCount, 1, "presenter should be called once")
XCTAssertEqual(presenterMock.interactorDidFetchBundleInfoReceivedArguments?.bundleInfo, bundleInfo, "interactor received wrong arguments")
```
Secondary, all test data(e.g `BundleInfo = ...`) should be in the `TestData` section:
```
extension AboutScreenInteractorTests {
    enum TestData {
        static let bundleInfo = ...
    }
}
```
due to this approach we will have more clear and legible code. 

Also I have never worked with `XCTest` framework thats why I don't know how it will be represented in the console output. I think we should pay attention to it too.